drift/inventory_service_interface.py

```diff
@@ -0,0 +1,25 @@
+import json
+import os
+import requests
+
+AUTH_HEADER_NAME = 'X-RH-IDENTITY'
+INVENTORY_SVC_URL = os.getenv('INVENTORY_SVC_URL', "http://inventory_svc_url_is_not_set")
+INVENTORY_SVC_PATH = '/r/insights/platform/inventory/api/v1'
+
+
+def get_key_from_headers(incoming_headers):
+    return incoming_headers.get(AUTH_HEADER_NAME)
+
+
+def fetch_hosts(host_ids, service_auth_key):
+    auth_header = {AUTH_HEADER_NAME: service_auth_key}
+    hosts = []
+
+    for host_id in host_ids:
+        response = requests.get(INVENTORY_SVC_URL + INVENTORY_SVC_PATH +
+                                '/hosts/' + host_id, headers=auth_header)
```

Comment:
The base URL with `hosts` path could be built outside of the list iteration and potentially stored as a constant above.

Reply:
updated